src/bp-forums/templates/default/js/editor.js

```diff
@@ -314,6 +323,18 @@ jQuery( document ).ready(
 				jQuery( window.forums_medium_topic_editor[ key ].elements[ 0 ] ).focus();
 				medium_editor.toggleClass( 'active' );
 			} );
+
+			jQuery( document ).on ( 'keyup', '#bbpress-forums .medium-editor-toolbar-input', function( event ) {
+
+				var URL = event.target.value;
+				
+				if ( bp.Nouveau.isURL( URL ) ) {
+					jQuery( event.target ).removeClass('isNotValid').addClass('isValid');
+				} else {
+					jQuery( event.target ).removeClass('isValid').addClass('isNotValid');
+				}
```

Comment:
@annrra

getting error  `JSHint: 'bp' is not defined.`

You need to define `bp` starting of the file.

like this

```
/* global bp, BP_Nouveau, Dropzone, videojs */
window.bp = window.bp || {};
```

Reply:
Thanks @chetansatasiya 
Done